content/neuvector-prime/azure/contents.lr

```diff
@@ -13,8 +13,17 @@ A step by step walk-through for creating a new offer from the Marketplace Overvi
 
 ## Basic (Project Details)
 
-1. Select existing subscription from the dropdown list. 
-2. Select existing Resource group from the dropdown list or create new. Provide name for new Resource group. 
+1. Select existing subscription from the dropdown list.
+2. Select existing Resource group from the dropdown list or create new Resource group.
+
+   Existing Resource Group selection
+
+   ![alt text](https://github.com/SUSE-Enceladus/marketplace-docs/tree/main/content/neuvector-prime/azure/images/existingresourcegroup.png?raw=true)
```

Comment:
I wonder, are hard-coded paths the way to go here... is there an alternative that uses relative paths to the images so that we can see how it looks prior to getting merged to main?

Reply:
Updated with relative paths.After it merges to azuredoc, hopefully we can see the image.